etna/records/tests/test_blocks.py

```diff
@@ -266,18 +268,22 @@ def test_add_featured_records(self):
             reverse("wagtailadmin_explore", args=(self.insights_index_page.id,)),
         )
         self.assertEqual(
-            self.insights_page.body.stream_data[0]["type"], "featured_records"
+            self.insights_page.body[0].block_type, "featured_records"
         )
+
         self.assertEqual(
-            self.insights_page.body.stream_data[0]["value"],
-            {
-                "heading": "This is a heading",
-                "introduction": "This is some text",
-                "records": ["C123456"],
-            },
+            self.insights_page.body[0].value["heading"],
+            "This is a heading"
         )
-
-        self.assertEqual(len(responses.calls), 2)
+        self.assertEqual(
+            self.insights_page.body[0].value["introduction"],
+            "This is some text"
+        )
+        self.assertEqual(
+            self.insights_page.body[0].value["records"][0].iaid,
+            "C123456"
+        )
+        self.assertEqual(len(responses.calls), 3)
```

Comment:
Why is there an extra API call now? Might be worth investigating

Reply:
The call for self.insights_page.body[0].value["record"].iaid , calls FetchQuery(self.model).get_multiple(iaid=iaid) , which is another KongClient fetch.